src/utilities/index.js

```diff
@@ -60,3 +60,13 @@ export {
 
 // Asserts that a string is a valid GraphQL name
 export { assertValidName } from './assertValidName';
+
+// Helpful functions to compare two GraphQLSchemas and detect breaking changes.
+export {
+  findBreakingChanges,
```

Comment:
I kept the other functions exported for a couple of reasons. One reason is so that the unit test can test them individually. The other is that other users might be interested in only a subset of the checks. This way they can pick and choose.